api/cpp/docs/mcu/stm32/generic.md

```diff
@@ -0,0 +1,41 @@
+<!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: MIT -->
+
+# Generic Instructions for Slint on STM32 MCUs
+
+The following instructions outline a rough, general path how to get started on an STM32 MCU
+with STM32 Cube tooling and Slint. Successful completion requires experience with STM32CubeMX
+as well as the peripherals of our board.
+
+1. Make sure to install all the <project:../stm32.md#prerequisites>.
+2. Start a new project with STM32CubeMX:
+   - Select your base board.
+   - Enable all peripherals needed. This includes LTDC and typically the FMC to be able
+     to place the framebuffers in RAM.
+   - Select CMake in the code generator options.
+3. General the CMake project and skeleton code.
+4. In the STM32 VS Code Extension, choose the command to import a CMake project.
+5. In STM32CubeMX select the STM32Cube BSP that matches your board and install it.
+6. Copy the BSP drivers into your project's source tree and modify `CMakeLists.txt` to
+   add them to the build.
+7. Add C++ support to the generated CMake project by adding `CXX` to the `LANGUAGES` option
+   of the `project` command.
+8. Download and extract <https://github.com/slint-ui/slint/releases/latest/download/Slint-cpp-nightly-thumbv7em-none-eabihf.tar.gz>
+9. Set `CMAKE_PREFIX_PATH` to the extracted directory.
+10. Adjust your `CMakeLists.txt` for Slint use:
+    1.  Add a `find_package(Slint)` call.
+    2.  Enable resource embedding by adding `set(DEFAULT_SLINT_EMBED_RESOURCES embed-for-software-renderer)`.
+    3.  Set the default style to `Fluent` with `set(SLINT_STYLE "fluent-light")`.
```

Comment:
Ditto. (Side note, maybe we should consider using material by default on embedded?)

Reply:
Ah yes, _this_ one we could pre-configure. I'll make a separate PR for that. I was also thinking Material, but I had issues with `Path`. Let's discuss that separately.

Reply:
Even better, we do have a default if freestanding. So this one is indeed easy to change. I think that was a leftover :)